### ISSUE TITLE
Replace trafaret by pydantic. it's much easier to use.

### ISSUE THREAD

**Issue by mrnicegyu11** [ITISFoundation/osparc-deployment-agent#5]
As suggested by @sanderegg

**Comment by pcrespov** [comment]
@mrnicegyu11 yes, we know :-) Perhaps you can also use the base classes and utils in https://github.com/ITISFoundation/osparc-simcore/tree/master/packages/service-library
